### PR TITLE
pull request to look at my prediction model 

### PR DESCRIPTION
**Pull request by ahuvabec**
my prediction model is called "Daily_bike_trips_prediction" and I by mistake stored it in my week1 folder

### CODE REVIEW

File: week1/Daily_bike_trips_prediction.Rmd [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+
+#  white backgrounds on our plots
+theme_set(theme_bw())
+options(repr.plot.width=4, repr.plot.height=3)
+
+
+# load the Data file 
+trips_per_day <- read_tsv('trips_per_day.tsv')
+holidays <- read.delim("holidays_2014.txt",header = FALSE, sep = ",", dec = ".")
```
Comment (jaredleekatzman) [inline week1/Daily_bike_trips_prediction.Rmd:25]: check out the package `library(Holidays)` - seems to provide a package for selecting US holidays! (saw other students use this)

File: week1/Daily_bike_trips_prediction.Rmd [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+  
+K <- 1:8 
+train_err = c()
+validation_err = c()
+
+#for loop to test each polynomial degree from 1-8 in our model
+for(k in K){
+  
+  model <- lm(num_trips ~ poly(tmax,k,raw = T)+I(did_rain)+I(is_weekend), data = train_trips_data)
```
Comment (jaredleekatzman) [inline week1/Daily_bike_trips_prediction.Rmd:93]: minor: could be good to add comments to explain the model and thought process in plain english. So if you return to this, you can remember why you did what you did!

File: week1/Daily_bike_trips_prediction.Rmd [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+#plotting the polynomial degree vs. error 
+ggplot(plot_data, aes(x = K, y = error, color = split))+
+  geom_line()+
+  scale_x_continuous(breaks = K)+
+  xlab('polynomial degree')+
+  ylab('RMSE')
+
+
+model <- lm(num_trips ~ poly(tmax,4, raw = T), data = train_trips_data)
```
Comment (jaredleekatzman) [inline week1/Daily_bike_trips_prediction.Rmd:114]: How did you choose poly4? Is this a baseline model? Or is this the best performing model?

**Review by jaredleekatzman — COMMENTED** [review]
Looks good! Ways to improve model and predictions could be adding some exploratory data analysis - looking at other trends you could add your model. 

Also any final take-aways about the model? Did it perform as expected? Better or worse? Documenting and discussing these are good ways to reflect and think about where you could go next with modeling. 

But overall structure looks good!

**Comment by jhofman** [comment]
Closing these dangling 2021 PRs out to reset for 2022
